### PR TITLE
translation for e10n

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e10n.js [hunk 1]
```diff
@@ -196,4 +207,91 @@ export default {
...
+        'Backward Shadow Implosion': 'バックワード・シャドウインプロージョン',
+        'Barbs Of Agony': 'バーブス・オブ・アゴニー',
+        'Cloak Of Shadows': 'クローク・オブ・シャドウ',
+        'Deepshadow Nova': 'ディープシャドウノヴァ',
+        'Distant Scream': '影の遠吠え',
+        'Fade To Shadow': '影潜り',
+        'Forward Implosion': 'フォワード・インプロージョン',
+        'Forward Shadow Implosion': 'フォワード・シャドウインプロージョン',
+        'Right Giga Slash': 'ライトサイド・ギガスラッシュ',
+        'Right Shadow Slash': 'ライトサイド・シャドウギガスラッシュ',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/raid/e10n.js:285]:
```suggestion
        'Left/Right Giga Slash': 'レフトサイド／ライトサイド・ギガスラッシュ',
        'Left/Right Shadow Slash': 'レフトサイド／ライトサイド・シャドウギガスラッシュ',
```

File: ui/raidboss/data/05-shb/raid/e10n.js [hunk 2]
```diff
@@ -196,4 +207,91 @@ export default {
...
+        'Spawn Shadow': 'Ombres croissantes',
+        'Throne Of Shadow': 'Trône de l\'Ombre',
+        'Umbra Smash': 'Fracas ombral',
+        'Void Pulse': 'Pulsation du néant',
+        'Voidgate': 'Porte du néant',
+      },
+    },
+    {
+      'locale': 'ja',
+      'missingTranslations': true,
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/raid/e10n.js:270]:
```suggestion
```

File: ui/raidboss/data/05-shb/raid/e10n.js (hunk 1 above)
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/raid/e10n.js:285]:
```suggestion
        'Forward Implosion': 'フォワード・インプロージョン',
        'Forward/Backward Implosion': 'フォワード／バックワード・インプロージョン',
        'Forward Shadow Implosion': 'フォワード・シャドウインプロージョン',
        'Front/Back Shadow Implosion': 'フォワード／バックワード・シャドウインプロージョン',
        'Left/Right Giga Slash': 'レフトサイド／ライトサイド・ギガスラッシュ',
        'Left/Right Shadow Slash': 'レフトサイド／ライトサイド・シャドウギガスラッシュ',
```

File: ui/raidboss/data/05-shb/raid/e10n.js [hunk 3]
```diff
@@ -196,4 +207,91 @@ export default {
...
+      'replaceSync': {
+        'Shadowkeeper': '影の王命',
+      },
+      'replaceText': {
+        'Backward Implosion': 'バックワード・インプロージョン',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/raid/e10n.js:275]:
```suggestion
        '(?<!/)Backward Implosion': 'バックワード・インプロージョン',
```

**Comment by quisquous** [comment]
@MaikoTan For what it's worth, I think translation additions to other people's patches are probably better done as follow-up pull requests rather than adding them in as comments on patches.

**Comment by MaikoTan** [comment]
Ah, ok i will pay attention next time.

**Pull request merged**